### PR TITLE
updating mongodb dependency

### PR DESCRIPTION
**Pull request by Juraci**
We updated mongodb dependency and tested against mongo 3.4 and 2.4.14
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/agco/harvesterjs/pull/209%23discussion_r108980904%22%2C%20%22https%3A//github.com/agco/harvesterjs/pull/209%23discussion_r108981519%22%2C%20%22https%3A//github.com/agco/harvesterjs/pull/209%23discussion_r108981829%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20d10dc18edb06f7100c13bcb2445bd369c0ceaf29%20package.json%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/agco/harvesterjs/pull/209%23discussion_r108980904%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22we%20are%20using%20the%20lib%20on%20version%202%2B%20and%20the%20mongo%20service%20on%203%2B.%20Should%20we%20update%20the%20mongdb%20lib%20version%20to%203%2B%20as%20well%3F%22%2C%20%22created_at%22%3A%20%222017-03-30T17%3A01%3A08Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/5835706%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/waldemarnt%22%7D%7D%2C%20%7B%22body%22%3A%20%22this%20is%20the%20latest%2C%20there%20is%20no%203.x.x%22%2C%20%22created_at%22%3A%20%222017-03-30T17%3A04%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/2266929%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Juraci%22%7D%7D%2C%20%7B%22body%22%3A%20%22Sorry%2C%20my%20fault%20xD%22%2C%20%22created_at%22%3A%20%222017-03-30T17%3A05%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/5835706%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/waldemarnt%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20package.json%3AL37-44%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull d10dc18edb06f7100c13bcb2445bd369c0ceaf29 package.json 5'></a> &nbsp;<b><a href='https://github.com/agco/harvesterjs/pull/209#discussion_r108980904'>File: package.json:L37-44</a></b>
- <a href='https://github.com/waldemarnt'></a> we are using the lib on version 2+ and the mongo service on 3+. Should we update the mongdb lib version to 3+ as well?
- <a href='https://github.com/Juraci'></a> this is the latest, there is no 3.x.x
- <a href='https://github.com/waldemarnt'></a> Sorry, my fault xD

<a href='https://www.codereviewhub.com/agco/harvesterjs/pull/209?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/agco/harvesterjs/pull/209?approve=1'></a>&nbsp;<a href='https://github.com/agco/harvesterjs/pull/209'></a>
<a href='#crh-end'></a>

---
This change is [Reviewable](https://reviewable.io/reviews/agco/harvesterjs/209)

***
***
***

### CODE REVIEW

**Comment by ssebro** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/agco/harvesterjs/pull/209'>Refresh</a>.<a href='#crh-update-none'></a>

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "lodash": "~3.7.0",
     "longjohn": "^0.2.4",
     "mkdirp": "~0.5.0",
-    "mongodb": "^1.4.26",
+    "mongodb": "^2.2.25",
```
Comment (waldemarnt) [inline package.json]: we are using the lib on version 2+ and the mongo service on 3+. Should we update the mongdb lib version to 3+ as well?

Comment (Juraci) [inline package.json]: this is the latest, there is no 3.x.x

Comment (waldemarnt) [inline package.json]: Sorry, my fault xD

**Pull request merged**